mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FaqScreen.kt

```diff
@@ -0,0 +1,49 @@
+package org.mifos.mobilewallet.mifospay.faq.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.designsystem.component.FaqItemScreen
+import org.mifos.mobilewallet.mifospay.designsystem.component.MifosTopBar
+import org.mifos.mobilewallet.mifospay.faq.presenter.FAQViewModel
+
+@Composable
+fun FaqScreen(
+    navigateBack: () -> Unit,
+    faqViewModel: FAQViewModel = hiltViewModel(),
+    updateFaqPosition: (Int) -> Unit,
```

Comment:
You don't need this anymore because you are handling in Faq Item screen using isSelected remember variable. So remove this.